### PR TITLE
Java cc6

### PR DESCRIPTION
**Pull request by wafman**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “append” - Expected outcome
     - [x] “insertBefore” - Expected outcome
     - [x] “insertAfter” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401Challenges/src/test/java/code401Challenges/linkedlist/LinkedListTest.java [hunk 1]
```diff
@@ -56,4 +53,82 @@ public void testPrint(){
         assertEquals(expectedOutput, classUnderTest.print());
     }
 
-}
\ No newline at end of file
+    @Test
+    public void testAppend(){
+        LinkedList classUnderTest = new LinkedList();
+        int num = 10;
+        classUnderTest.insert(15);
```
Comment (evanslaton) [inline code401Challenges/src/test/java/code401Challenges/linkedlist/LinkedListTest.java]: Instead of using `insert` and then only using `append` at the end, it would have been better to just use `append`
